pandas/tests/io/test_excel.py

```diff
@@ -1862,6 +1862,62 @@ def test_invalid_columns(self):
             with pytest.raises(KeyError):
                 write_frame.to_excel(path, 'test1', columns=['C', 'D'])
 
+    def test_comment_arg(self):
+        # Test the comment argument functionality to read_excel
+        with ensure_clean(self.ext) as path:
+
+            # Create file to read in
+            write_frame = DataFrame({'A': ['one', '#one', 'one'],
+                                     'B': ['two', 'two', '#two']})
+            write_frame.to_excel(path, 'test_c')
+
+            # Read file without comment arg
+            read_frame = read_excel(path, 'test_c')
+            read_frame_commented = read_excel(path, 'test_c', comment='#')
+            tm.assert_class_equal(read_frame, read_frame_commented)
```

Comment:
be consistent with other tests on nameing, e.g. 
``write_frame -> df``
``result = read_excel``


use ``assert_frame_equal``